_posts/fr/2018-06-11-donner-aux-utilisateurs-les-moyens-de-réussir.html

```diff
@@ -0,0 +1,121 @@
+---
+title: "Donner aux utilisateurs les moyens de réussir grâce à la conception de contenu"
+description:
+author: Zak Kain, concepteur de contenu
+date: 2018-06-11 09:00:00 -0400
```

Comment:
again, you should change the date to the post date (annoying because you were waiting on a review sorry :s)